core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -754,6 +793,41 @@ describe('Contributions and review component', function() {
       });
     });
 
+    it('should fetch skill when user clicks on ' +
+      'view suggestion', fakeAsync(function() {
+      spyOn($uibModal, 'open').and.returnValue({
+        result: Promise.resolve([])
+      });
+      spyOn(contributionAndReviewService, 'resolveSuggestiontoSkill')
+        .and.callFake((
+            targetId, suggestionId, action, reviewMessage,
+            skillDifficulty, resolveSuggestion, cb) => {
+          resolveSuggestion();
+          cb();
+        });
```

Comment:
Add a new line here to separate the setup from the action.

Reply:
done.